[2023-12-05T21:54:35.838+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_openaq_catchup.load_measures_data manual__2023-12-05T21:54:32.342401+00:00 [queued]>
[2023-12-05T21:54:35.842+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_openaq_catchup.load_measures_data manual__2023-12-05T21:54:32.342401+00:00 [queued]>
[2023-12-05T21:54:35.842+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-12-05T21:54:35.848+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_measures_data> on 2023-12-05 21:54:32.342401+00:00
[2023-12-05T21:54:35.852+0000] {standard_task_runner.py:57} INFO - Started process 398 to run task
[2023-12-05T21:54:35.856+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_openaq_catchup', 'load_measures_data', 'manual__2023-12-05T21:54:32.342401+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/dag_openaq_catchup.py', '--cfg-path', '/tmp/tmp3iyhmm_b']
[2023-12-05T21:54:35.859+0000] {standard_task_runner.py:85} INFO - Job 276: Subtask load_measures_data
[2023-12-05T21:54:35.899+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_openaq_catchup.load_measures_data manual__2023-12-05T21:54:32.342401+00:00 [running]> on host 01a38cb1bd44
[2023-12-05T21:54:35.955+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_openaq_catchup' AIRFLOW_CTX_TASK_ID='load_measures_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-05T21:54:32.342401+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-05T21:54:32.342401+00:00'
[2023-12-05T21:54:35.957+0000] {main.py:140} INFO - Archivo de configuracion leido correctamente.
[2023-12-05T21:54:35.957+0000] {utils.py:25} INFO - Obteniendo datos de https://api.openaq.org/v2/measurements...
[2023-12-05T21:54:36.709+0000] {utils.py:73} INFO - Conexi√≥n a la base de datos establecida exitosamente.
[2023-12-05T21:54:39.349+0000] {utils.py:90} INFO - Cargando datos en la tabla stg_measurements...
[2023-12-05T21:54:41.192+0000] {utils.py:98} INFO - Datos cargados exitosamente en la tabla stg_measurements.
[2023-12-05T21:54:41.196+0000] {main.py:166} INFO - Ejecutando query CALL nicolasmvinciguerra_coderhouse.petl_load_fact_measure();...
[2023-12-05T21:54:41.991+0000] {main.py:168} INFO - Se ejecuto correctamente la query: CALL nicolasmvinciguerra_coderhouse.petl_load_fact_measure();
[2023-12-05T21:54:42.278+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-05T21:54:42.308+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_openaq_catchup, task_id=load_measures_data, execution_date=20231205T215432, start_date=20231205T215435, end_date=20231205T215442
[2023-12-05T21:54:42.355+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-05T21:54:42.371+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
